circuits/aggregator/prover.go

```diff
@@ -0,0 +1,40 @@
+package aggregator
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/consensys/gnark-crypto/ecc"
+	"github.com/consensys/gnark/backend/groth16"
+	"github.com/consensys/gnark/frontend"
+)
+
+// Prove method of AggregatorCircuit instance generates a proof of the
+// validity values of the current assignment. It loads the required circuit
+// artifacts and decodes them to the proper format. It returns the proof or an
+// error.
+func (assignment AggregatorCircuit) Prove() (groth16.Proof, error) {
+	// load circuit artifacts content
+	if err := Artifacts.LoadAll(); err != nil {
+		return nil, fmt.Errorf("failed to load vote verifier artifacts: %w", err)
+	}
+	// decode the circuit definition (constrain system)
+	ccs := groth16.NewCS(ecc.BW6_761)
```

Comment:
We have some constants for the curves of each circuit, here you can use `circuits.AggregatorCurve`. Please, update also these variables in the `voteverifier.Prove` method.

Reply:
Point me those constants please